### PR TITLE
LPS-172418 Relationship Tab is not exported properly

### PR DESCRIPTION
**Pull request by leilanyaragao**
Related issue: [LPS-172418](https://issues.liferay.com/browse/LPS-172418)

### CODE REVIEW

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by leilanyaragao** [comment]
ci:test:object

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java [hunk 1]
```diff
@@ -586,7 +586,7 @@ public void testAddOrUpdateSystemObjectDefinition() throws Exception {
 
 					@Override
```
Comment (seltong) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java]: @leilanyaragao could you run the SF in this commit, please? You removed some tabs by mistake.
Please, use this [PR](https://github.com/liferay-objects/liferay-portal/pull/2239) as example.

File: modules/apps/object/object-api/src/main/java/com/liferay/object/service/ObjectRelationshipLocalService.java [hunk 2]
```diff
@@ -409,4 +409,11 @@ public ObjectRelationship updateObjectRelationship(
 			Map<Locale, String> labelMap)
 		throws PortalException;
```
Comment (seltong) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/service/ObjectRelationshipLocalService.java:410]: could you squash this commit with the previous buildService?

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 3]
```diff
@@ -198,7 +198,7 @@ public void testAddObjectRelationship() throws Exception {
 				ObjectRelationshipConstants.TYPE_MANY_TO_MANY +
 					" does not allow a parameter object field ID",
 			() -> _objectRelationshipLocalService.addObjectRelationship(
-				TestPropsValues.getUserId(),
+				null, TestPropsValues.getUserId(),
```
Comment (seltong) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:201]: Could you move these changes to [this commit](https://github.com/liferay-objects/liferay-portal/pull/2788/commits/46df3bdf012bffe02d75018df9481b5757ab14a1)? 
You can follow [this example](https://github.com/liferay-objects/liferay-portal/pull/2239/commits/f2019700d6aae4a8436c35b03979834b68ef58b0) from my PR

**Review by seltong — CHANGES_REQUESTED** [review]

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by leilanyaragao** [comment]
ci:test:object

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:object

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
This PR must wait for #2819 to be merged.

**Pull request closed without merging**